src/task/active.ts

```diff
@@ -289,7 +297,11 @@ export default class ActiveTaskImpl<T> extends EventEmitter
             ...this._finishPatches.scheduleNextRun(
               nextRunDelayMs !== undefined
                 ? nextRunDelayMs + this._data.timestamp()
-                : computeNextRun(this.interval, this._processingStart)
+                : computeNextRun(
```

Comment:
i see you're not taking end time into account for "non-terminal" acknowledgements (i.e. defer, retry, release). i think that's reasonable, but can you add a note to the option's documentation mentioning that it will not prevent retries/defers/releases on an existing run of a task? that way we don't have to remember that/look at the source code in the future :)